.github/workflows/benchmark.yml

```diff
@@ -23,11 +23,11 @@ jobs:
     if: github.event_name != 'pull_request' || contains(github.event.pull_request.labels.*.name, 'benchmark')
     steps:
       - uses: actions/checkout@v3
-      - name: Set up JDK 17
+      - name: Set up JDK 21
         uses: actions/setup-java@v3
         with:
-          java-version: '17'
-          distribution: 'temurin'
+          java-version: '21'
+          distribution: 'corretto'
```

Comment:
We initially used `temurin` due to build caches, but since we use `corretto` in production it's not awful to build & test against that.